src/cozmo/robot.py

```diff
@@ -1141,6 +1172,28 @@ def go_to_pose(self, pose, relative_to_robot=False):
         self._action_dispatcher._send_single_action(action)
         return action
 
+    def go_to_object(self, object, distance_from_object):
+        '''Tells Cozmo to drive to the specified object.
+
+        Args:
+            object (:class:`cozmo.objects.ObservableObject`): The destination object.
+            distance_from_object (:class:`cozmo.util.Distance`): The distance from
```

Comment:
Maybe reword the distance_from_object description like this:
'The distance from the object to stop. This is the distance between the origins. For instance, the distance from the robot's origin (between Cozmo's two front wheels) to the cube's origin (at the center of the cube) is ~40mm.